src/main.js

```diff
@@ -43,6 +44,14 @@ const urlParams = getURLParams(window)
 const { store, router, apolloProvider } = init(urlParams)
 const { SplashScreen, StatusBar } = Plugins
 
+// Mobile Intercom
+if (config.mobileApp) {
+  const intercom = new Intercom()
+  const userId = "lunie-app-" + Math.floor(Math.random() * 10000 + 1).toString()
+  intercom.registerIdentifiedUser({ userId })
+  Vue.prototype.$mobileIntercom = intercom
```

Comment:
ooooook, better would have been to pass it to the store and trigger an event from the component to do something in the store. but this probably works

Reply:
I can do it!